deploy/overlays/dev-template/config.yaml

```diff
@@ -19,3 +19,4 @@ spec:
     url: ${GIT_DEPLOY_URL}
     disableSSLVerification: ${GIT_DISABLE_SSL_VERIFICATION}
   requireArtifactVerification: true
+  hermeticBuilds: Required
```

Comment:
Given its eventually evaluated to a boolean in createPipelineSpec would it be better to make it a boolean here and in jbsconfig_types?